apis/pom.xml

```diff
@@ -60,13 +60,13 @@
     <!-- Quarkus -->
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-    <quarkus.platform.version>3.6.1</quarkus.platform.version>
+    <quarkus.platform.version>3.11.3</quarkus.platform.version>
```

Comment:
It looks like 3.12 is released now (https://quarkus.io/blog/quarkus-3-12-0-released/). Can we update to the latest?

Reply:
Sure I can try that.

Reply:
Updated, waiting for CI re-run.

Reply:
CI looks good after upgrade.